projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/facade/NodeFactoryFacade.java

```diff
@@ -77,6 +77,14 @@
     // also update {@link NodeFactoryFacadeComposite} when changing or removing this name
     void setUpFacade(LLVMParserRuntime runtime);
 
+    /**
+     * Get Parser runtime (a temporary required method until we use the new Bitcode Type exclusive).
+     *
+     * @return runtime
+     */
+    @Deprecated
```

Comment:
I do not think that we can remove the parser runtime. We just need to transition to the new types. Can we remove the `@Deprecated` warning or is there an issue that I overlooked?